CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+- Fixed broken delegation for global meter providers. (#5827)
+  Changed the reflect.TypeOf() to use a nil pointer so we aren't allocating on the heap unless we need to.
```

Comment:
Please move it to `Fixed` section.

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/5828/commits/8b2c127853258cc3c988fc07529b839e919e89ff